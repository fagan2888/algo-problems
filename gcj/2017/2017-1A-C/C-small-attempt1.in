100
11 5 16 5 0 0
3 1 3 2 2 0
3 1 3 2 1 0
2 1 5 1 1 1
92 1 96 45 0 1
95 1 60 55 4 21
90 1 100 44 0 1
93 1 97 46 0 1
79 1 41 50 4 3
94 1 92 46 0 1
45 2 48 27 4 7
65 1 58 38 4 4
100 1 25 54 0 4
92 1 100 47 0 1
96 1 91 49 0 1
98 1 99 48 0 1
97 1 98 49 0 1
91 1 22 47 0 4
98 1 100 48 0 1
8 1 39 11 2 6
98 1 97 50 0 1
24 10 47 10 13 9
94 20 77 55 4 1
99 1 90 49 0 1
97 1 49 48 0 2
92 1 30 45 0 3
76 1 30 30 3 1
97 1 99 49 0 1
39 1 82 42 2 30
100 1 32 50 0 3
90 1 91 44 0 1
93 1 100 46 0 1
25 1 55 10 1 6
95 1 23 47 0 4
88 7 48 60 0 0
99 1 95 48 0 1
91 1 99 54 4 27
30 1 98 20 4 5
90 1 96 45 0 1
97 1 32 49 0 3
46 1 79 22 3 11
100 1 32 49 0 3
48 1 26 25 2 2
90 1 95 44 0 1
58 1 69 92 3 51
100 1 97 51 0 1
30 1 48 15 1 1
92 1 19 23 0 0
97 1 49 50 0 2
96 1 92 47 0 1
43 1 38 38 2 32
69 3 88 56 15 3
92 1 97 46 0 1
38 1 76 17 4 4
51 1 49 76 2 36
95 4 23 34 0 0
96 1 91 48 0 1
28 5 50 7 0 0
100 1 23 50 0 4
91 1 92 46 0 1
93 1 24 47 0 4
90 1 97 44 0 1
97 1 47 50 0 2
91 1 94 45 0 1
26 1 60 13 3 2
93 1 94 46 0 1
91 1 48 74 1 15
96 1 22 47 0 4
96 1 31 48 0 3
100 1 95 49 0 1
96 1 90 48 0 1
90 1 97 45 0 1
97 1 25 52 0 4
91 1 96 44 0 1
50 1 94 3 3 3
77 1 20 37 1 6
25 1 34 68 3 20
93 1 96 47 0 1
99 2 2 8 0 0
99 1 92 48 0 1
100 1 30 52 0 3
99 1 94 49 0 1
45 1 13 26 2 3
80 15 55 89 33 87
95 1 48 49 0 2
98 1 30 50 0 3
87 1 56 40 3 9
96 1 46 49 0 2
36 41 85 76 28 49
98 1 95 49 0 1
96 1 92 48 0 1
93 1 98 45 0 1
94 1 46 47 0 2
92 1 97 45 0 1
98 1 95 50 0 1
98 1 98 48 0 1
47 1 97 20 4 4
93 1 49 47 0 2
96 1 90 47 0 1
91 1 66 40 2 2
