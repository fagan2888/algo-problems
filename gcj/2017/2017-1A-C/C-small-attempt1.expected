Case #1: 5
Case #2: 2
Case #3: IMPOSSIBLE
Case #4: 5
Case #5: 161
Case #6: 11
Case #7: 164
Case #8: 164
Case #9: IMPOSSIBLE
Case #10: 158
Case #11: 11
Case #12: 17
Case #13: 44
Case #14: 169
Case #15: 163
Case #16: 169
Case #17: 170
Case #18: 38
Case #19: 170
Case #20: 11
Case #21: 171
Case #22: 5
Case #23: IMPOSSIBLE
Case #24: 161
Case #25: 83
Case #26: 50
Case #27: 9
Case #28: 171
Case #29: 15
Case #30: 55
Case #31: 155
Case #32: 167
Case #33: 16
Case #34: 39
Case #35: IMPOSSIBLE
Case #36: 165
Case #37: 12
Case #38: 15
Case #39: 162
Case #40: 55
Case #41: 12
Case #42: 55
Case #43: 13
Case #44: 159
Case #45: 12
Case #46: 172
Case #47: 25
Case #48: 27
Case #49: 85
Case #50: 160
Case #51: 11
Case #52: IMPOSSIBLE
Case #53: 164
Case #54: 13
Case #55: 13
Case #56: 9
Case #57: 161
Case #58: 13
Case #59: 40
Case #60: 159
Case #61: 40
Case #62: 161
Case #63: 83
Case #64: 159
Case #65: 16
Case #66: 161
Case #67: IMPOSSIBLE
Case #68: 37
Case #69: 53
Case #70: 166
Case #71: 160
Case #72: 163
Case #73: 43
Case #74: 159
Case #75: 11
Case #76: 13
Case #77: IMPOSSIBLE
Case #78: 165
Case #79: 1
Case #80: 162
Case #81: 54
Case #82: 166
Case #83: 11
Case #84: 4
Case #85: 83
Case #86: 53
Case #87: 14
Case #88: 81
Case #89: 5
Case #90: 167
Case #91: 162
Case #92: 163
Case #93: 79
Case #94: 162
Case #95: 169
Case #96: 168
Case #97: 15
Case #98: 83
Case #99: 158
Case #100: 19
